app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,20 @@
+class MoviesController < ApplicationController
+
+  def show
+    @movie = Movie.find(params[:id])
+    @actor_names = @movie.order_by_age
```

Comment:
It's okay to call @movie.order_by_age in the view so the instance vars on line 5 and 6 are not strictly necessary!